working-groups/wg-dm-portal.md

```diff
@@ -0,0 +1,22 @@
+# DM Portal Working Group
+
+## Goals
+
+Build a small web app to help generate the DM config file easier.
```

Comment:
do we have any design doc about shit web app?

Reply:
yep, let me find it out and attach the link.

Reply:
@zz-jason , added them below, PTAL